projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -1065,3 +1098,17 @@ export function login(email: string, password: string, name: string): void {
   // namely the logged in user should be greeted
   cy.get('.cx-login-greet').should('contain', name);
 }
+
+/**
+ * Clicks nth group menu link
+ *
+ * @param {number} index
+ * @returns {Chainable<JQuery<HTMLElement>>}
+ */
+export function clickOnNthGroupMenu(
```

Comment:
There is a method **clickOnGroup** that could be used for the navigation within the configuration via the group menu.
Please, have a look at it and use this method accordingly. It is unnecessary to have 2 methods for the same behaviour.